addons/medical/functions/fnc_clearFinished.sqf

```diff
@@ -0,0 +1,34 @@
+/*
+ * Author: Orbis2358
+ * Remove finished actions from list
+ *
+ * Arguments:
+ * 0: The medic <OBJECT>
+ * 1: The patient <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [medic, patient] call ace_medical_fnc_clearFinished
+ *
+ * Public: Yes
+ */
+
+#include "script_component.hpp"
+
+params ["_caller", "_target"];
+
+private _joint = _target getVariable [QGVAR(jointTreatment), []];
+private _itemCount = 0;
+
+if !(_joint isEqualTo []) then {
```

Comment:
This if statement is logically unnecessary, performance wise I doubt if this is better. Is anyone able to comment on this? 

Even so, given the amount of times this would be executed, any performance impact would be negligible, so I'd opt for simplifying the logic here.